### PR TITLE
@joeyAghion => Make sure not found artists and artworks bubble up as 404s

### PR DESCRIPTION
**Pull request by mzikherman**
I suspect a better solution exists in Reaction but that's...going to take me a while to wade thru and figure out, so this can be a quick fix for now.

### CODE REVIEW

File: src/desktop/apps/artist/server.tsx [hunk 1]
```diff
@@ -90,7 +90,13 @@ app.get(
       res.status(status).send(layout)
     } catch (error) {
       console.log(error)
-      next(error)
+      if (error.message.includes("Artist Not Found")) {
```
Comment (mzikherman) [inline src/desktop/apps/artist/server.tsx]: Theoretically there could be _other_ errors here, of which 'Artist Not Found' is only one.

However, this is the error handler for the server-side route handler, so we're definitely not serving a 2xx page already at this point. So assuming its a 404 and returning 'Artist Not Found' is _probably_ semi reasonable, especially if it's just a quick fix while a better one is worked out in Reaction.

File: src/desktop/apps/artwork2/server.tsx [hunk 2]
```diff
@@ -65,7 +65,13 @@ app.get(
       res.status(status).send(layout)
     } catch (error) {
       console.log(error)
-      next(error)
+      if (error.message.includes("Artwork Not Found")) {
```
Comment (mzikherman) [inline src/desktop/apps/artwork2/server.tsx]: Same comment as the above one.

**Comment by joeyAghion** [comment]
Thanks for the quick fix. Since this seems fragile and is untested, though, my inclination is to wait for a better fix closer to where the error is happening (reaction, if you say so). The current state isn't harming user experience in a significant way at the moment.

**Comment by mzikherman** [comment]
Re-opening as many 500s instead of 404s obscure error traces and may be hiding _actual_ 500s and issues.

It also seems (cc @alloy ) that this approach (along w/ some nuance like checking there's only one error present), as well as tests, might be suitable?

**Pull request merged**